package.json

```diff
@@ -25,6 +25,7 @@
     "@commitlint/config-conventional": "^17.0.3",
     "@lavamoat/allow-scripts": "^3.0.0",
     "@lavamoat/preinstall-always-fail": "^2.0.0",
+    "@types/react": "18.0.15",
```

Comment:
what is the reason to add this special version? if this is cause by type error, why other snap does not have this issue?

Reply:
let's remove this.